_posts/2019-05-09-How-We-Release-Code.md

```diff
@@ -0,0 +1,96 @@
+- **To set expectations for the maintainers**, such as amount of time needed to maintain the project, sign-off from the developers lead and ensuring the project does not require Zalando-specific dependencies
+
+![Project Release Form](/assets/img/os-release-form.png)
+
+You can see a public version of the [approval form without validation here](https://goo.gl/forms/9C4xlel5DlIK52Xw1).
+
+Questions addressing who have signed-off on the publishing and how many hours the developers have been granted to maintain the project serve as a good way to set expectations, both to the lead who sign off and for the maintainers. To run a sustainable project requires time and resources and we do not expect developers to use their private time, instead granting work time should be part of the conversation. 
+
+We also address the need to have basic project health files in place such as Code of conduct, ways for users to get in touch in case of security issues and to reach out to the maintainers - we provide maintainers with a [boiler-plate set of files](https://github.com/zalando-incubator/new-project) for guidance. We do this for 2 reasons:
+
+1. Ownership of code should be visible to other teams inside Zalando, and to potential audits, beyond ownership, these files also communicate how to contribute, how to report security issues and our code of conduct. 
+2. Communication channels should be public, so maintainers of a project must be approachable by external contributors, we do not ever intent to just throw code over the wall, so having clear ways to reach our maintainers is a central part of taking active ownership of code. 
+
+
+## The Open Source Review Group
+When a project is proposed, it is automatically shared on an internal mailing-list which consists of everyone at Zalando currently maintaining an approved project - which is currently a group of around 200 people - this structure allows us to spread the decision making process across many different people and viewpoints. 
+
+![Review Group](/assets/img/review-group.png)
+
+
+### Discussing the why
+The point here is to have as many eyes on the proposal as possible, specifically we are interested in discussing the WHY of releasing a project and the 3 questions below is central in this discussion:
+
+1. Can the project be sustainable?
+2. Do Zalando have any value in taking ownership?
+3. Does it have any value to anyone outside Zalando?
+
+When source code is released as open source, you are essentially giving away something of value, you are also taking responsibility for spending time and resources on the additional overhead associated with open sourcing, this loss of value and use of resources should be justified, there are multiple ways to look at this, such as:
+
+1. The project contributes positively to the employer branding efforts and supports hiring of tech talent
```

Comment:
This is stated as a requirement, but can't be proven before release. How is this measured and what input and expectations come from EP?